lib/dma.c

```diff
@@ -463,7 +475,24 @@ _dma_addr_sg_split(const dma_controller_t *dma,
     return cnt;
 }
 
-int dma_controller_dirty_page_logging_start(dma_controller_t *dma, size_t pgsize)
+static void
+dirty_mapped_sg(dma_controller_t *dma)
```

Comment:
nit, this sounds like a boolean, but it's actively dirtying all mapped sg's, can we make this be "dma_mark_dirty_sgs" or somesuch

Reply:
done